calico-enterprise/operations/selinux.mdx

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
I think this would be better placed under `getting-started/install-on-clusters/`. 

It's best to get this information directly to users as part of the install process. I'd suggest we add this as either an optional step or a prerequisite to every relevant installation procedure.

Reply:
I'll hold specific edits on this section until after we've settled on a location.